src/pudl/transform/eia176.py

```diff
@@ -0,0 +1,113 @@
+"""Module to perform data cleaning functions on EIA176 data tables."""
+
+import pandas as pd
+from dagster import AssetCheckResult, AssetIn, AssetOut, asset_check, multi_asset
+
+from pudl.logging_helpers import get_logger
+
+logger = get_logger(__name__)
+
+
+@multi_asset(
+    outs={
+        "core_eia176__yearly_company_data": AssetOut(),
+        "core_eia861__yearly_aggregate_data": AssetOut(),
+    },
+)
+def _core_eia176__data(
+    raw_eia176__data: pd.DataFrame,
+) -> tuple[pd.DataFrame, pd.DataFrame]:
+    """Take raw list and return two wide tables with primary keys and one column per variable.
+
+    One table with data for each year and company, one with state- and US-level aggregates per year.
+    """
+    raw_eia176__data = raw_eia176__data.astype({"report_year": int, "value": float})
+    raw_eia176__data["variable_name"] = (
+        raw_eia176__data["line"] + "_" + raw_eia176__data["atype"]
+    )
+
+    aggregate_primary_key = ["report_year", "area"]
+    company_primary_key = aggregate_primary_key + ["id"]
+    company_drop_columns = ["itemsort", "item", "atype", "line", "company"]
+    # We must drop 'id' here and cannot use as primary key because its arbitrary/duplicate in aggregate records
+    # 'id' is a reliable ID only in the context of granular company data
+    aggregate_drop_columns = company_drop_columns + ["id"]
+
+    long_company = raw_eia176__data.loc[
+        raw_eia176__data.company != "total of all companies"
+    ]
+    wide_company = get_wide_table(
+        long_table=long_company.drop(columns=company_drop_columns),
+        primary_key=company_primary_key,
+    )
+
+    long_aggregate = raw_eia176__data.loc[
+        raw_eia176__data.company == "total of all companies"
+    ]
+    wide_aggregate = get_wide_table(
+        long_table=long_aggregate.drop(columns=aggregate_drop_columns),
+        primary_key=aggregate_primary_key,
+    )
+
+    return wide_company, wide_aggregate
```

Comment:
Do these not need to be wrapped in Dagster `Output` definitions to correctly identify which corresponds to what output asset? That's what we've done in all the other `@multi_asset` functions. E.g. in `pudl.transform.eia861`